deploy/osd-fedramp-machineconfig/99-fedramp-chrony.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: machineconfiguration.openshift.io/v1
```

Comment:
Can you separate out these yaml files into their respective manifests? 00-fedramp-chrony-worker.yaml and 00-fedramp-chrony-master.yaml